src/common/promises/symbols.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the MIT license found in the
+ * LICENSE file in the root of this projects source tree.
+ */
+
+export const $$toStringTag: symbol = Symbol("toStringTag");
```

Comment:
Could you explain what this is doing?

Reply:
Yes, the symbol can guarantee there would be only one single reference of the symbol value at the runtime.
And we could use the symbol as the index key to return a cancelTokenTag string, that tag string could be used to tell us whether one cancellation token instance is one of us customized cancel token instance or not. 
By using the symbol key, we would avoid naming collision, there won't any risks of other types of cancellation tokens carrying similar tags and confused us. 😬